openshift/topics/introduction.adoc

```diff
@@ -1,35 +1,66 @@
+These templates use OpenShift's internal link:{ocpdocs_serving_x509_secrets_link}[service serving x509 certificate secrets] to automatically create the HTTPS keystore used for serving secure content. The JGroups cluster traffic is authenticated using the `AUTH` protocol and encrypted using the `ASYM_ENCRYPT` protocol. The {project_name} server truststore is also created automatically, containing the */var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt* CA certificate file, which is used to sign the certificate for HTTPS keystore.
+
+Moreover, the truststore for the {project_name} server is pre-populated with the all known, trusted CA certificate files found in the Java system path. These templates secure the TLS communication using re-encryption TLS termination.
+
+* _{project_templates_version}-x509-https_*: {project_name} {project_version} with auto-generated HTTPS keystore and {project_name} truststore, backed by internal H2 database. The `ASYM_ENCRYPT` JGroups protocol is used for encryption of cluster traffic.
+* _{project_templates_version}-x509-postgresql-persistent_*: {project_name} {project_version} with auto-generated HTTPS keystore and {project_name} truststore, backed by persistent PostgreSQL database. The `ASYM_ENCRYPT` JGroups protocol is used for encryption of cluster traffic.
+
+==== Other templates
 
 Other templates that integrate with {project_name} are also available:
 
-* *_eap64-sso-s2i_*: {project_name}-enabled Red Hat JBoss Enterprise Application Platform 6.4.
-* *_eap71-sso-s2i_*: {project_name}-enabled Red Hat JBoss Enterprise Application Platform 7.1.
-* *_datavirt63-secure-s2i_*: {project_name}-enabled Red Hat JBoss Data Virtualization 6.3.
+* _eap64-sso-s2i_*: {project_name}-enabled Red Hat JBoss Enterprise Application Platform 6.4.
+* _eap71-sso-s2i_*: {project_name}-enabled Red Hat JBoss Enterprise Application Platform 7.1.
+* _datavirt63-secure-s2i_*: {project_name}-enabled Red Hat JBoss Data Virtualization 6.3.
 
 These templates contain environment variables specific to {project_name} that enable automatic {project_name} client registration when deployed.
 
-See xref:Auto-Man-Client-Reg[Automatic and Manual {project_name} Client Registration Methods] for more information.
+[role="_additional-resources"]
+.Additional resources
+
+* xref:Auto-Man-Client-Reg[Automatic and Manual {project_name} Client Registration Methods]
+* link:{ocp311docs_passthrough_route_link}[passthrough TLS termination]
+* link:{ocp311docs_reencrypt_route_link}[re-encryption TLS termination]
+
+=== Version Compatibility and Support
+For details about OpenShift image version compatibility, see the xPaaS part of the https://access.redhat.com/articles/2176281[OpenShift and Atomic Platform Tested Integrations page].
```

Comment:
The first sentence (and mainly the `https://access.redhat.com/articles/2176281` link) should be dropped as outdated IMHO. The https://access.redhat.com/articles/2176281 doesn't mention recent RH-SSO 7.4 / 7.5 streams, and corresponding OCP 4.x tested integrations pages:
* https://access.redhat.com/articles/4128421
* https://access.redhat.com/articles/4763741
doesn't mention RH-SSO at all.

Maybe we should instead point the users to RH-SSO tested / supported integrations page:
* https://access.redhat.com/articles/2342861
for version compatibility & support information?